gpytorch/kernels/scale_kernel.py

```diff
@@ -94,11 +94,12 @@ def forward(self, x1, x2, batch_dims=None, diag=False, **params):
             outputscales = outputscales.unsqueeze(1).repeat(1, x1.size(-1)).view(-1)
 
         orig_output = self.base_kernel.forward(x1, x2, diag=diag, batch_dims=batch_dims, **params)
-
         if torch.is_tensor(orig_output):
             outputscales = outputscales.view(-1, *([1] * (orig_output.dim() - 1)))
 
         if diag:
             return delazify(orig_output) * outputscales
-
         return orig_output.mul(outputscales)
+
+    def size(self, x1, x2):
```

Comment:
We might want to add this to our other decorator kernels as well (GridInterpolationKernel, ProductKernel, etc.)